documentation/assemblies/deploying/assembly-deploy-tasks.adoc

```diff
@@ -3,25 +3,27 @@
 // deploying/deploying.adoc
 
 [id="deploy-tasks_{context}"]
-= Deploying Strimzi
+= Deploying Strimzi using installation artifacts
 
-Having xref:deploy-tasks-prereqs_{context}[prepared your environment for a deployment of Strimzi], this section shows:
+[role="_abstract"]
+Having xref:deploy-tasks-prereqs_{context}[prepared your environment for a deployment of Strimzi], you can deploy Strimzi to a Kubernetes cluster.
+You can use the deployment files provided with the release artifacts.
 
-* xref:deploy-create-cluster_{context}[How to create the Kafka cluster]
-* Optional procedures to deploy other Kafka components according to your requirements:
-** xref:kafka-connect-{context}[Kafka Connect]
-** xref:kafka-mirror-maker-{context}[Kafka MirrorMaker]
-** xref:kafka-bridge-{context}[Kafka Bridge]
+Use the deployment files to xref:deploy-create-cluster_{context}[create the Kafka cluster].
 
-The procedures assume a Kubernetes cluster is available and running.
+Optionally, you can deploy the following Kafka components according to your requirements:
+
+* xref:kafka-connect-{context}[Kafka Connect]
+* xref:kafka-mirror-maker-{context}[Kafka MirrorMaker]
```

Comment:
Pardon my ignorance, but why we do not mention MM2?

Reply:
The bullet links to a section that tells you how to install 1 or 2